templates/repo/settings/options.tmpl

```diff
@@ -122,7 +122,7 @@
 							<tbody>
 								<tr>
 									<td colspan="4">
-										<div class="text red gt-py-4 gt-border-secondary-bottom">{{ctx.Locale.Tr "repo.settings.mirror_settings.direction.pull"}}: {{ctx.Locale.Tr "error.occurred"}}</div>
+										<div class="text red gt-py-4 tw-border tw-border-secondary-bottom">{{ctx.Locale.Tr "repo.settings.mirror_settings.direction.pull"}}: {{ctx.Locale.Tr "error.occurred"}}</div>
```

Comment:
IIRC, "gt-border-secondary-bottom" was only defined by us? Does "tw-" have it?

Reply:
Tailwind has all our colors as per config, including `secondary`.

Reply:
Really? It isn't there on my side.

![image](https://github.com/go-gitea/gitea/assets/2114189/dceea5c1-150a-42bc-8932-8caea41baff3)

Reply:
Ah, I think it should be `tw-border-b-secondary`, will try.

Reply:
fixed, `gt-border-secondary-bottom` is equivalent to `tw-border-b tw-border-b-secondary`. It's a bit of a shame that tailwind can not do colored border with a single class.